### ISSUE TITLE
Piped data always overrides arguments

### ISSUE THREAD

**Issue by bjan-sag** [reubenmiller/go-c8y-cli#209]
I believe this is a bug:
```
$ echo "dummy" | c8y tenants get --id management
2023-01-17T12:55:36.832+0100	ERROR	serverError: There is no tenant with id = dummy GET http://management.acme.com/tenant/tenants/dummy: 404 security/Not Found There is no tenant with id = dummy
```

I'd expect it to fetch the management tenant as I specify id directly.

I had a similar issue with `applications` but I forgot the exact command and I can't reproduce it.

**Comment by reubenmiller** [comment]
Yes that looks to be a bug,  I think it might be limited commands using the `tenant` as the iterator. The correct behaviour should definitely be that the "management" value is used for the `--id` flag.

Previously this problem was also affecting some application commands but I think that was fixed a while ago. Though obviously if you have an example that would help.

**Comment by reubenmiller** [comment]
Fixed in v2.20.2

**Comment by bjan-sag** [comment]
Yeah, I tried some more ways of breaking `applications` and it's all been working fine :) So I guess I had an old version then. Thank you!
